DEPS

```diff
@@ -11,17 +11,25 @@ Core dependencies
 
 Essential:
 
-* python (== 2.7.x)
+* python3 (>= 3.6)
 * bsddb (distributed with stdlib) or bsddb3
 * gtk+ >= 3.10
 * gstreamer >= 1.4
 * gstreamer-plugins-base >= 1.6
 * gstreamer-plugins-good >= 1.4
-* mutagen (>= 1.10)
-* dbus-python
-* pygobject (>= 3.13.2)
-* python-cairo
+* python3-mutagen (>= 1.10)
 * GI typelib files for GTK+, GStreamer (including gstreamer-plugins-base) and cairo (python-gi-cairo on Ubuntu)
+  * Packages on Debian and Ubuntu:
+    * python3-gi
+    * python3-gi-cairo
+    * gir1.2-gtk-3.0
+    * gir1.2-gstreamer-1.0
+    * gir1.2-gst-plugins-base-1.0
+  * Packages on Fedora:
+    * python3-cairo
+    * python3-dbus
```

Comment:
dbus-python is not a GI library, so it shouldn't be under this section.

Reply:
You are right, thanks!